docs/eventing/broker/_index.md

```diff
@@ -1,7 +1,27 @@
 ---
-title: "Broker"
+title: "Brokers"
 weight: 60
 type: "docs"
+showlandingtoc: "false"
 ---
 
-{{% readfile file="README.md" %}}
+Brokers are Kubernetes [custom resources](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) that define an event mesh for collecting a pool of [CloudEvents](https://cloudevents.io/), and provide a discoverable endpoint, `status.address`, for event delivery. Event producers can send events to a broker by POSTing the event to the `status.address.url` of the broker.
```

Comment:
> and provide a discoverable endpoint, `status.address`, for event delivery.

reads a little misleading?  Maybe:

> and provide a discoverable endpoint (`status.address`) for event ingress, and Triggers for event delivery.

Reply:
Updated. I won't add parentheses if it's not necessary because it's not best practice in technical writing - the idea is that if something is important it goes in the main sentence, if it's not, take it out entirely. New commit coming soon to show the change.